### PR TITLE
feat(3): added 2 lambdas

### PR DESCRIPTION
**Pull request by dartwlad**
1. Services is done, FE is working. https://d2igrds53f0okp.cloudfront.net
Additional tasks:
Async/await is used in lambda functions;
Typescript (so and ES6 modules);
Webpack is configured for product-service;
Unit tests (Jest);
Attempt to split handlers to separate module and integrate with principles of 3-tier architecture (handlers, services, data);
Adding error handling on a base handler level;
Add serverless-aws-documentation plugin and npm script to download generated Openapi documentation (Swagger).
2. https://2bs26yvgma.execute-api.us-east-1.amazonaws.com/dev/products
    https://2bs26yvgma.execute-api.us-east-1.amazonaws.com/dev/products/1
3. FE PR https://github.com/dartwlad/nodejs-aws-fe/pull/2
4. Documentation file is committed.

### CODE REVIEW

**Review by IDiamondDragon — APPROVED** [review]
Task 3 was done. Score: 11

File: products-service/serverless.ts [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  functions: {
+    getProducts: {
+      handler: 'handler.getProducts',
+      events: [
+        {
+          http: {
+            method: 'get',
+            path: 'products',
+            cors: true,
+            request: {
```
Comment (petr150894) [inline products-service/serverless.ts:107]: it seems this request param configuration should be moved to getProductsById lambda config block

**Review by petr150894 — APPROVED** [review]
Well done!

File: products-service/serverless.ts (hunk 1 above)
Comment (dartwlad) [reply products-service/serverless.ts:107]: You are right. Thanks. Will fix.

**Pull request merged**